django/applications/catmaid/tests/apis/test_links.py

```diff
@@ -21,7 +21,7 @@ def test_delete_link_failure(self):
                  'state': make_nocheck_state()})
         self.assertEqual(response.status_code, 200)
         parsed_response = json.loads(response.content.decode('utf-8'))
-        expected_result = {'error': 'Couldn\'t find link between connector {0} and node {0}'.format(connector_id)}
+        expected_result = {'error': 'Couldn\'t find link between connector {0} and node {0}'.format(connector_id)} # FIXME the repeated {0} is weird and probably a bug
```

Comment:
Please tell me how to fix this

Reply:
It looks like second just needs to be `treenode_id` (which is defined earlier); this test passes because in this one instance, `treenode_id == connector_id`.